fractions/jaeger/src/main/java/org/wildfly/swarm/jaeger/JaegerFraction.java

```diff
@@ -0,0 +1,84 @@
+package org.wildfly.swarm.jaeger;
+
+import org.jboss.logging.Logger;
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+
+import static com.uber.jaeger.Configuration.*;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@DeploymentModule(name = "org.wildfly.swarm.jaeger", slot = "deployment")
```

Comment:
I kindly ask the Swarm developers to review this whole dependency thing for this PR... I have to admit that I don't really know how it works, and couldn't find a documentation about it, so, I kept trying things I've seen from other modules until I got a setup that works.